networks/_modules/ignite/main.tf

```diff
@@ -1,3 +1,12 @@
+terraform {
+  required_providers {
+    quorum = {
+      source = "ConsenSysQuorum/quorum"
```

Comment:
@trung @nmvalera now can we update to `Consensys/quorum`?

Reply:
:+1: pushed